schema/models_directory/dft/tags.json

```diff
@@ -0,0 +1,14 @@
+{
+    "allOf": [
+        {
+            "$ref": "../../model/tags.json"
+        }
+    ],
+    "description": "Tags specific to this model type only.",
+    "items": {
+        "enum": [
+            "excited-states",
+            "scaling-degree:3"
```

Comment:
(1) These might not be specific to dft only. Shall we move them to the definitions file (perhaps adding a description there too)?
(2) there must be multiple levels of scaling